packages/embedded-techdocs-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "embedded-techdocs-app",
-  "version": "0.0.0",
```

Comment:
this is an interesting one, I don't think it will release this package anyways (looking at the main.yml workflow https://github.com/backstage/techdocs-cli/blob/main/.github/workflows/main.yml#L7.), but maybe we don't have do even change version here. What do you think?

Reply:
I agree with you as it's an internal dependency we don't need to increment the version